source/_data/glossary.yml

```diff
@@ -271,8 +271,25 @@
   link: /docs/blueprint/selectors/
 
 - term: Service
-  definition: >-
-    Services are called to perform actions.
+  definition: |-
+    The term service has 2 meanings in Home Assistant:
+
+    **The information service**
+    E.g. the municipal waste management service that provides entities for organic, paper, and packaging waste. In terms of functionality, the information service is a device. It is called *service* to avoid confusion as it does not come with a piece of hardware.
+
+    **The software function that interacts with targets to make something happen**
+    A service carries out one specific task: e.g. turn on the light in the living room or send a notification to a mobile phone. 
```

Comment:
One thing that goes through my mind: A service can do something, as in, it is an available option. Calling a service would actually do that.

The "calling" isn't really made clear or explained (and worse, one of my suggestions in this review removes the single occurrence of it 🙃).

Reply:
Maybe we can keep that in mind for a second iteration?